tests/setuidgid/Makefile.BSD

```diff
@@ -17,6 +17,6 @@ SRCS	+=	warnp.c
 IDIRS	+=	-I${LIBCPERCIVA_DIR}/util
 
 test:	test_setuidgid
-	./test_setuidgid.sh
+	./test-setuidgid-minimal.sh
```

Comment:
Why not just run the single command here?  Now that we're using awk to extract the test: target there's no issue with multiply escaping `$` characters.